Containers/apache/Dockerfile

```diff
@@ -1,59 +1,60 @@
 # Caddy is a requirement
 FROM caddy:2.6.2-alpine as caddy
 
-FROM debian:bullseye-20230109-slim
+FROM httpd:2.4.54-alpine3.17
+
+RUN set -ex; \
+    apk add --no-cache shadow; \
+    groupmod -g 333 xfs; \
+    usermod -u 333 -g 333 xfs; \
+    addgroup -g 33 -S www-data; \
+    adduser -u 33 -D -S -G www-data www-data ; 
+    apk del --no-cache shadow
 
 RUN mkdir -p /mnt/data; \
     chown www-data:www-data /mnt/data;
 
 VOLUME /mnt/data
 
 RUN set -ex; \
-    \
-    apt-get update; \
-    apt-get install -y --no-install-recommends \
-        apache2 \
+    apk add --no-cache \
+        bash \
         supervisor \
         wget \
+        tzdata \
         ca-certificates \
         openssl \
-        netcat \
-        dpkg-dev \
-    ; \
-    rm -rf /var/lib/apt/lists/*
+        netcat-openbsd
 
 COPY --from=caddy /usr/bin/caddy /usr/bin/
 RUN chmod +x /usr/bin/caddy
 
-RUN a2enmod rewrite \
-    headers \
-    proxy \
-    proxy_fcgi \
-    setenvif \
-    env \
-    mime \
-    dir \
-    authz_core \
-    alias
-
-COPY nextcloud.conf /etc/apache2/sites-available/
-
-RUN rm /etc/apache2/ports.conf; \
-    sed -s -i -e "s/Include ports.conf//" /etc/apache2/apache2.conf; \
-    sed -i "/^Listen /d" /etc/apache2/apache2.conf
+RUN sed -i \
+		-e 's/^#\(LoadModule .*mod_rewrite.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_headers.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_proxy.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_proxy_fcgi.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_setenvif.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_env.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_mime.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_dir.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_authz_core.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_alias.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_mpm_event.so\)/\1/' \
+		-e 's/\(LoadModule .*mod_mpm_worker.so\)/#\1/' \
+		-e 's/\(LoadModule .*mod_mpm_prefork.so\)/#\1/' \
+		conf/httpd.conf
+
+COPY nextcloud.conf conf
+RUN echo Include conf/nextcloud.conf | tee -a conf/httpd.conf
+
+RUN sed -i "/^Listen /d" conf/httpd.conf
+echo "ServerName localhost" | tee -a conf/httpd.conf
```

Comment:
same as https://github.com/nextcloud/all-in-one/pull/1577/files#r1085253864 and below

Reply:
done